### ISSUE TITLE
Performance evaluation?

### ISSUE THREAD

**Issue by cmaimone** [freelawproject/eyecite#93]
RE: https://github.com/openjournals/joss-reviews/issues/3617

There aren't performance claims in the paper, but I think maybe there should be -- in the paper or in the tutorial.  If I'm evaluating whether to use this package, I have no idea how good it is.  Is there a standard corpus (or can you make one?) where you can run your code and report some basic stats on how many citations you extract, how many errors, etc?  It doesn't even have to be that big of a set of documents -- but some indication of the types of text/documents you've tested against, and how well it does on those.  

I get that this package may be the best available for the task, but I have no idea what that means in practice -- can I rely on the results, or does it miss a lot?

**Comment by mlissner** [comment]
We do say this in the readme:

> eyecite is an open source tool for extracting legal citations from text. It is used, among other things, to annotate millions of legal documents in the collections of CourtListener and the Caselaw Access Project.

It's kind of hard to count errors, because we have fixed pretty much all of the possible citations. I guess we could say something like, "We don't know of any citations that it is unable to parse."

**Comment by cmaimone** [comment]
Yes - even something clearer along those lines.  We've parsed XXX documents in these repositories and reviewed XXX by hand to ensure that eyecite found all citations in these XXX documents.  Just something to let people know the type of performance they can expect
